incubator/prometheus/templates/_helpers.tpl

```diff
@@ -0,0 +1,10 @@
+{{/*
+Create a default fully qualified app name.
+
+We truncate at 24 chars because some Kubernetes name fields are limited to this
+(by the DNS naming spec).
+*/}}
+{{define "fullname"}}
```

Comment:
@prydonius so far every chart has this template. Is there a way to bake it into helm somehow?